manager/rbac_manager.py

```diff
@@ -134,33 +136,33 @@ def format_permissions(res):
         return res
 
     @staticmethod
-    def need_permissions(perms: [RbacPermission], operator="or"):
+    def need_permissions(perms: [RbacPermission], operator="and"):
```

Comment:
I've changed the default operator to be AND since we request inventory read everywhere